### ISSUE TITLE
Zip packaged with Gradle is not valid

### ISSUE THREAD

**Issue by jiyuan312986471** [saagie/gradle-saagie-plugin#4]
Hi, 

I'm trying packaging my jobs from Saagie platform using Gradle.

I installed Gradle by following the official instructions for Windows, edited _build.gradle_, _gradle.properties_ and then run the command to package jobs.

From console output, all things went well. But when I checked the zips packaged, **they were not valid**.

Here's the _build.gradle_:
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath group: 'io.saagie', name: 'gradle-saagie-plugin', version: '2.+'
    }
}

apply plugin: 'io.saagie.gradle-saagie-plugin'

saagie {
    packaging {
	exportFile = 'test'
	currentOnly = true
    }
    target = './delivery'
    server {
	acceptSelfSigned = true
	url = 'https://manager.saagie2.matmut.int/api/v1'
	login = 'myLogin'
	password = 'myPwd'
	platform = 2
    }
    jobs {[
	{
            id = 1370
        }
    ]}
}
```

And the command to launch the packaging:
`gradle --full-stacktrace --info exportJob`

I could get a zip file: _1370-test.zip_ but it's not valid.
Windows error message when opening the zip: 
![image](https://user-images.githubusercontent.com/9774401/41025999-a4249bde-6973-11e8-89e7-9f8e654bda99.png)

I've tried with Gradle 4.7 and Gradle 4.1 and they all get the same error.

Do you have any hint?

Thanks in advance

**Comment by erwan-koffi** [comment]
Do you have the possibility to open the generated zip on an *nix machine?

**Comment by erwan-koffi** [comment]
Correction released in 2.1.2
